tests/parametric/README.md

```diff
@@ -1 +1 @@
-Full documentation can be found in [docs/scenarios/parametric.md](https://github.com/DataDog/system-tests/blob/main/docs/scenarios/parametric.md).
+Full documentation can be found in [docs/scenarios/parametric.md](https://github.com/DataDog/system-tests/blob/main/docs/scenarios/parametric.md).
```

Comment:
what changed here? newline at the end of the file?

could we omit this change from the PR?

Reply:
I've been fighting git with this file. Nothing is different, but yet it's still showing a delta. I tried running `git checkout origin/main -- parametric/README.md` locally to revert any changes, but that's not doing anything either. I'm open to advice 🤦

Reply:
0_o

Reply:
Ah, I was running `git checkout` with the wrong README file. Should be gone now.